monkey/agent_plugins/payloads/cryptominer/config-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "properties": {
+        "simulation_time": {
+            "title": "Simulation time",
+            "type": "number",
+            "description": "The time (in seconds) for which the cryptomining simulation should run on each machine",
+            "default": 300,
+            "minimum": 0
+        },
+        "cpu_utilization": {
+            "title": "CPU utilization",
+            "type": "integer",
+            "description": "The percentage of CPU to use on a machine",
+            "default": 20,
```

Comment:
Maybe a more realistic default would be good? I think cryptominers often use 100%. 80% seems close but not dangerous. Thoughts?

Reply:
Sounds good.